src/Libraries/CoreNodeModels/Selection.cs

```diff
@@ -56,6 +60,28 @@ private set
             }
         }
 
+        [JsonProperty("NodeType")]
+        public string NodeType
+        {
+            get
+            {
+                return "ExtensionNode";
+            }
+        }
+
+        [JsonProperty("InstanceId")]
+        public IEnumerable<string> SelectionIdentifier
+        {
+            get
+            {
+                return selection.Select(GetIdentifierFromModelObject).Where(x => x != null).ToList();
+            }
+            set
+            {
+                selectionIdentifier = value;
+            }
+        }
```

Comment:
I have tried several ways of setting the Revit element ID from the constructor but none have worked.  This currently, isn't doing anything.